Resources/Prototypes/XenoArch/Effects/normal_effects.yml

```diff
@@ -495,3 +495,10 @@
     maxSpawns: 1
     spawns:
     - id: Singularity
+
+- type: artifactEffect
+  id: EffectEmp
+  targetDepth: 2
```

Comment:
all of the effects are loosely sorted from low to high depth. At the very least, put this one in the section with the other depth 2 artifacts. Alternatively, bump it up to depth 3.